.env.template

```
@@ -35,4 +35,16 @@ JWT_REFRESH_TOKEN_EXPIRATION=
+
+# Frontend Configuration
+FRONTEND_BASE_URL=
+FRONTEND_DEVELOP_URL=
+
+# NCP Configuration
+NCP_ACCESS_KEY=
+NCP_SECRET_KEY=
+NCP_BUCKET_NAME=
+
+# Sentry
+SENTRY_DSN=
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The SENTRY_DSN key is duplicated

(DuplicatedKey)